README.md

```diff
@@ -7,6 +8,7 @@ There will eventually be a suggested order of completion, but at this time since
 1. Reverse String
 
 ## HOW TO USE THESE EXERCISES
+## READ THIS CAREFULLY 
```

Comment:
![image](https://user-images.githubusercontent.com/4411701/95520528-1a3a5000-098d-11eb-9734-9422b3ac4b86.png)


When using two headers, the markdown is not displayed properly

Reply:
Instead of using two headers, one can just make a new **BOLD** paragraph under the header